src/Kdyby/Validator/DI/ValidatorExtension.php

```diff
@@ -52,16 +54,18 @@ public function loadConfiguration()
 			->setFactory('Symfony\Component\Validator\Mapping\Loader\AnnotationLoader')
 			->addTag(self::TAG_LOADER);
 
+		$cacheService = $builder->addDefinition($this->prefix('cache'))
+			->setClass('Symfony\Component\Validator\Mapping\Cache\CacheInterface');
+
 		$cacheFactory = self::filterArgs($config['cache']);
-		if (!class_exists($cacheFactory[0]->entity) || !in_array('Symfony\Component\Validator\Mapping\Cache\CacheInterface', class_implements($cacheFactory[0]->entity), TRUE)) {
-			throw new Nette\Utils\AssertionException(
-				'Expected implementation of Symfony\Component\Validator\Mapping\Cache\CacheInterface, ' .
-				'but ' . $cacheFactory[0]->entity  . ' given.'
-			);
+		if (class_exists($cacheFactory[0]->entity) && in_array('Symfony\Component\Validator\Mapping\Cache\CacheInterface', class_implements($cacheFactory[0]->entity), TRUE)) {
+			$cacheService->setFactory($cacheFactory[0]->entity, $cacheFactory[0]->arguments);
+		} else {
+			$cacheService->setFactory('Symfony\Component\Validator\Mapping\Cache\DoctrineCache', array(
```

Comment:
I would rather implement a bridge for symfony cache like doctrine-cache is, but without the actuall doctrine